image/image_linux.go

```diff
@@ -394,6 +405,15 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 			return err
 		}
 
+		// Its a bit more tricky to deal with local snaps, as we only have that specific revision
+		// available, unless we switch and download the revision provided by the seed.manifest. However
+		// we should not alter the expected outcome, instead lets fail and inform the user.
```

Comment:
Depends on the meaning of 'existing behaviour', because this new behaviour is related to the reproducible build. The point was that we should not silently correct the user, instead we fail and let the user correct their behaviour. Does this make sense? I'll happily correct it if the wording is misleading

Reply:
Yes, it's a bit unclear then: the first sentence seems to imply that the correct behaviour would be downloading the exact revision mentioned in the manifest (and according to your reply here, it seems that this would be indeed be the correct thing to do), but the second sentence seems to justify why we are not doing it, and this gets me confused.

What is the correct behaviour?